bundles/org.openhab.automation.pwm/src/main/java/org/openhab/automation/pwm/internal/handler/PWMTriggerHandler.java

```diff
@@ -242,11 +247,6 @@ public void dispose() {
             localEventSubscriberRegistration.unregister();
         }
 
-        StateMachine localStateMachine = stateMachine;
-        if (localStateMachine != null) {
-            localStateMachine.stop();
```

Comment:
Just to be sure that this is deliberately no more needed

Reply:
Yes, that caused the exception.